packages/css/src/details.css

```diff
@@ -8,11 +8,13 @@
   --dsc-details-padding: var(--ds-size-2) var(--ds-size-4);
   --dsc-details-size: var(--ds-size-14);
   --dsc-details-background: var(--ds-color-background-default);
+  --dsc-details-color: var(--ds-color-text-default);
   --dsc-details-heading-background--hover: var(--ds-color-surface-tinted);
   --dsc-details-heading-background--open: var(--ds-color-background-tinted);
   --dsc-details-heading-background: var(--ds-color-background-default);
 
   background: var(--dsc-details-background);
+  color: var(--dsc-details-color);
```

Comment:
Why not just inherit color? What use-case do we need this?

Reply:
This makes sure it uses the correct color from the `data-color` set